tests/foreman/longrun/test_oscap.py

```diff
@@ -598,11 +700,15 @@ def test_positive_oscap_run_via_local_files(
 
     :CaseImportance: Critical
     """
+    contenthost = rex_contenthost
+    os_version = contenthost.os_version.major
+    distro = f'rhel{os_version}'
     SELECTED_ROLE = 'theforeman.foreman_scap_client'
     file_name = 'security-data-oval-com.redhat.rhsa-RHEL8.xml.bz2'
     download_url = 'https://www.redhat.com/security/data/oval/v2/RHEL8/rhel-8.oval.xml.bz2'
```

Comment:
It may be out of scope, but it would be nice if we also prepared this for RHEL 9 (as you already did with a lot of other aspects) by introducing more constants.

Reply:
noted, and it makes sense. But I'd revisit this in a different PR, as we plan to add more stuff here